packages/test-utils/lib/test-utils.js

```diff
@@ -6,7 +6,6 @@ const { Keystone } = require('@keystonejs/keystone');
 const { GraphQLApp } = require('@keystonejs/app-graphql');
 const { KnexAdapter } = require('@keystonejs/adapter-knex');
 const { MongooseAdapter } = require('@keystonejs/adapter-mongoose');
-const { runCustomQuery } = require('@keystonejs/server-side-graphql-client');
```

Comment:
I believe this means we can remove the package dependency on `server-side-graphql-client`